.project

```diff
@@ -30,5 +30,14 @@
 				<arguments>1.0-name-matches-false-false-nbproject</arguments>
 			</matcher>
 		</filter>
+		<filter>
+			<id>1599077280157</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
```

Comment:
What does this do? I'm not at all against it, just curious.

Reply:
Good question - it was added, seemingly, by the Java language server - probably to not process `.git` and `node_modules` folders...